test/unit/empty.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
????? 🤔

Reply:
I've had never seen such performance improvement, brilliant. 😆

Reply:
It is for `svg`. https://github.com/verdaccio/verdaccio/blob/4a6383d05f90814c081bcab9e229638dc51f7bfc/jest.config.js#L45 I tried `svg` with identity-obj-proxy but facing some errors.

Reply:
How is the trick?

Reply:
` '\\.(svg)$': '<rootDir>/test/unit/empty.js'` in jest.config.js